### ISSUE TITLE
support for cyclone 0.4

### ISSUE THREAD

**Issue by gilbertohasnofb** [agraef/purr-data#10]
There has been a recent new release of the `cyclone` library, now version 0.4. Version 0.3 was a major overhaul and fixed a lot of the annoying bugs that previous versions had. 

Could you consider supporting version 0.4 and perhaps integrating it directly into Purr-Data instead of shipping it with an older version?

See: https://github.com/porres/pd-cyclone/releases

**Comment by gilbertohasnofb** [comment]
Some additional info: https://github.com/porres/pd-cyclone/issues/466#issuecomment-554113811

**Comment by porres** [comment]
I guess this is the best place to open issues for Purr Data: https://git.purrdata.net/jwilkes/purr-data/issues

**Comment by gilbertohasnofb** [comment]
@porres yes, that is a good idea. I opened an issue over there. Many thanks, Alexandre.

**Comment by agraef** [comment]
Unearthing an old ticket after reopening the tracker on GH. Gee, how time passes, this was originally submitted when we didn't even know about Covid yet. ;-)

I'm leaving this issue open because, as recently discussed on the DISIS mailing list, we still haven't ported Alexandre's latest version yet. This probably wouldn't be too difficult, if the comment object was removed from it and the help patches. But nobody volunteered to do that so far, so I guess that we'll have to wait and stick to the old cyclone for the time being.

OTOH, there's an initial port of Alexandre's ELSE, see https://github.com/agraef/purr-data/releases/tag/2.19.2+ELSE. This is still experimental, though (in particular, the gui features haven't been ported yet), but the non-gui objects are supposed to work, so they might be a suitable replacement.
